command/cp.go

```diff
@@ -966,3 +1012,57 @@ func guessContentType(file *os.File) string {
 	}
 	return contentType
 }
+
+type CountingWriter struct {
+	pb progressbar.ProgressBar
+	fp *os.File
+}
+
+func (r *CountingWriter) WriteAt(p []byte, off int64) (int, error) {
+	n, err := r.fp.Write(p)
```

Comment:
r.fp.Write or r.fp.WriteAt?

Reply:
Also, you could write it as

```go
n, err := r.fp.Write(p)
r.pb.AddCompletedBytes(int64(n))
return n, err
```

Reply:
Should we increment the progress bar if write fails?